src/coreclr/vm/inlinetracking.h

```diff
@@ -26,7 +26,7 @@
 #include "corhdr.h"
 #include "shash.h"
 #include "sarray.h"
-#include "crsttypes.h"
+#include "crsttypes_generated.h"
```

Comment:
Can this `#include "crst.h"` ? It looks odd for this one file to be including `crsttypes_generated.h` directly.

Reply:
Hmm, we are very inconsistent with casing. I guess it can stay as is.